### PR TITLE
Edit candles to be slightly more realistic

### PR DESCRIPTION
**Pull request by Lumi-Virtual**
#### Summary
Balance "Makes the candle's properties and recipe slightly more realistic"

#### Purpose of change
As I was going through items to draw ASCII art for I realized the candle had a few values that were funky (and the density for paraffin wax could be more precise). It felt like too many changes to justify including in the ASCII art pr so here it is separately.

#### Describe the solution
Just changes some values to be more accurate to what I was able to find online, and makes the recipe for candles more expensive so that mass isn't being created out of nothing like it was previously. Now when you make a candle you only get about 93% of the input mass instead of 172%

#### Describe alternatives you've considered
I could have used a different candle as reference, but the candle it's based on right now is a fairly standard size, and the total weight and volume are close to a multiple of the individual crafting component items, like paraffin and bees wax which are both 250 ml per unit item.

#### Testing
I compiled from a recent master pull and the recipe and all the numbers looked right.

### CODE REVIEW

**Comment by Lumi-Virtual** [comment]
not sure if this is a balance or a bugfix change, but since nothing was really broken before i assume its balance ?

**Comment by anothersimulacrum** [comment]
> not sure if this is a balance or a bugfix change, but since nothing was really broken before i assume its balance ?

Doesn't really matter, it'll get cut out of the changelog anyways.

**Comment by EternalLaughter** [comment]
Heh, for realism all the candles should be changed to be x-packs of candles as well. When was the last time you bought a single candle and not a big pack to throw in the cupboard?

**Comment by I-am-Erk** [comment]
Definitely worth having, feel free to post as an issue

**Comment by EternalLaughter** [comment]
> Definitely worth having, feel free to post as an issue

Agreed...so i looked into it, but it seems the candles we're using are the really big chonky bois as they last 35 hours and is almost half a kilo (0.47)

These types of candles i have zero idea how many and often people buy, and it makes more sense they are rarer, compared to a pack of "regular" thin stick sized candles that last like 8'ish hours, or a pack of 100 pack of tea-candles at 6 hours.

Basically we would need a new type of candle for the packs to make sense, and i'm not sure it would be preferable over just the big chonky ones we got in use now.  It would feel redundant, even if it's more fitting for a household.

**Comment by I-am-Erk** [comment]
Again, this really should be in an issue. It would be fine to make these into "votive candles" and have a pr adding ten-packs of "taper candles"
